dallinger/frontend/static/scripts/dallinger2.js

```diff
@@ -734,4 +734,4 @@ try {
   module.exports.dallinger = dallinger;
 } catch (err) {
   // We aren't being loaded from a node context, no need to export
-}
\ No newline at end of file
+}
```

Comment:
Normally best to keep the blank line at the end of the file but it's not a big deal.

Reply:
Actually, what I did here is adding that blank line.